### PR TITLE
Add grouping and expandible

### PR DESCRIPTION
**Pull request by vicpada**
Added support for:
* group by city
* expand description

### CODE REVIEW

**Comment by vicpada** [comment]
This PR resolves #1 and #2

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+jQuery(document).ready(function($){
+
+    $('.lever .lever-job-title').click( function() {
```
Comment (juancarrey) [inline js/main.js]: this could be `$('.lever').on('click', '.lever-job-title', function() {...`

**Comment by juancarrey** [comment]
LGTM

File: js/main.js (hunk 1 above)
Comment (vicpada) [reply js/main.js]: done

**Pull request merged**